globals/utils/email.py

```diff
@@ -0,0 +1,21 @@
+from django.conf import settings
+from django.core.mail import send_mail
+
+
+# TODO: make this function asynchronous
+# TODO: add logging on success/error of the sending email
+def send_async_mail(subject, message, recipients):
```

Comment:
Please rename the function just `send_mail`  
We will use this method in `tasks.py`, which is celeries standard way to define async tasks.

Reply:
Sorry not required.

Reply:
Done!